src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import com.google.sps.data.ResponseStatus;
+
+/* Servlet that handles adding review and rating to entity */
+@WebServlet("/add_review")
+public class ReviewServlet extends HttpServlet {
+
+    private static final String PROFILE_ID = "ProfileID";
+    private static final String REVIEW_ENTITY = "ReviewEntity";
+    private static final String PROFILE_ID_PROPERTY = "ProfileID";
+    private static final String ENTITY_TYPE_PROPERTY = "EntityType";
+    private static final String ENTITY_ID_PROPERTY = "EntityID";
+    private static final String ENTITY_RATING_PROPERTY = "Rating";
+    private static final String ENTITY_COMMENT_PROPERTY = "Comment";
+    private static final String TIMESTAMP_PROPERTY = "timestamp";
```

Comment:
looks like the same constants everywhere...can you please create a common/Constants.java and use that everywhere?